client/packages/system/src/Clinician/ListView/ListView.tsx

```diff
@@ -29,6 +29,7 @@ export const ClinicianListView = () => {
     isError,
     isLoading: isFetching,
     enableRowSelection: false,
+    enableColumnFilters: false,
```

Comment:
Oh, I was thinking of some internal logic that checks the column definitions and turns this off if there are no filters enabled, so we don't have to explicitly add this to our table definition. Is that tricky to do?

Reply:
Ohh, maybe not, I'll have a quick look in my current PR